test/sagemaker_tests/tensorflow/inference/test/integration/local/test_multi_model_endpoint.py

```diff
@@ -80,6 +80,7 @@ def test_ping():
 
 
 @pytest.mark.skip_gpu
+@pytest.mark.model("half_plus_three")
```

Comment:
nit: Missing pytest.mark.processor("cpu")

Reply:
good catch, thanks

Reply:
if skip_gpu is added, is it fair to say processor is just cpu? or will eia also run?

Reply:
synced offline